eacl/example_test.go

```diff
@@ -1,29 +1,17 @@
 package eacl_test
 
 import (
-	"fmt"
-
-	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
 	"github.com/nspcc-dev/neofs-sdk-go/container/acl"
 	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
-	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+	"github.com/nspcc-dev/neofs-sdk-go/crypto/test"
 	"github.com/nspcc-dev/neofs-sdk-go/eacl"
 	"github.com/nspcc-dev/neofs-sdk-go/object"
 )
 
-func randomPublicKey() neofscrypto.PublicKey {
-	k, err := keys.NewPrivateKey()
-	if err != nil {
-		panic(fmt.Errorf("randomize private key: %v", err))
-	}
-
-	return neofsecdsa.Signer(k.PrivateKey).Public()
-}
-
 // eACL provides ability to determine target subjects of access rules, in
 // particular, distribute/restrict access on an exclusive basis.
 func ExampleTable_exclusiveRights() {
-	friendPubKey := randomPublicKey()
+	friendPubKey := test.RandomPublicKey()
```

Comment:
NACK. It's an example. It must not import any test packages, it should be something people can copy/paste. Sure, `randomPublicKey()` will need to be replaced, but at least one can see what's in there, unlike when it's an imported package.